src/components/Extensions/Details/DetailCard/Tabs/EventsConfig.test.tsx

```diff
@@ -24,21 +24,44 @@ describe('EventsConfig', () => {
   it('Renders channel and allowed events config options', () => {
     const displayName = 'Pirojok-the-extension';
     const id = 'pirojok';
-    const POST_INSTALL_CONTENT = 'CONTENT';
+    const postInstallContent = 'CONTENT';
+    const eventarcChannel =
+      'projects/test-project/locations/us-central1/channels/firebase';
+    const allowedEventTypes = ['google.firebase.v1.custom-event-occurred'];
     const extension: Extension = {
       id,
       displayName,
-      postinstallContent: POST_INSTALL_CONTENT,
-      eventarcChannel:
-        'projects/test-project/locations/us-central1/channels/firebase',
-      allowedEventTypes: ['google.firebase.v1.custom-event-occurred'],
+      postinstallContent: postInstallContent,
+      eventarcChannel,
+      allowedEventTypes,
     } as Extension;
 
-    const { getByText } = render(
+    const { queryByTestId, getByText } = render(
       <TestExtensionsProvider extensions={[extension]} instanceId={id}>
         <EventsConfig />
       </TestExtensionsProvider>
     );
+    expect(queryByTestId('allowed-event-types-config')).not.toBeNull();
```

Comment:
Can we test that the rendered content includes the events? Instead of data-test-id